src/cartesian/getTicks.ts

```diff
@@ -24,9 +24,11 @@ function getTicksEnd(
 
   for (let i = len - 1; i >= 0; i--) {
     let entry = result[i];
-    const size = getTickSize(entry, i);
+    let size: number | undefined;
+    const getSize = () => size ?? getTickSize(entry, i);
```

Comment:
This, I do not understand. When would this ever return size? When would size not be undefined?

Reply:
Sorry, I made a mistake here. It was a leftover from some refactors I made.
Thanks, updated 👍